CLI-workflow-executor/README.md

```diff
@@ -0,0 +1,88 @@
+Note: This is a clone of Pablo Moreno's EPI group repository [https://github.com/ebi-gene-expression-group/galaxy-workflow-executor](https://github.com/ebi-gene-expression-group/galaxy-workflow-executor)
```

Comment:
It's not the group of Pablo.

Reply:
Why is this clone needed at all?

Reply:
Because customized setting file would be needed and also makes it independent of upstream changes. Alternatively I can fork the repo and modify the fork as a separate repo. Would you prefer that?